src/shadow-embed.js

```diff
@@ -158,35 +163,6 @@ export function getShadowRootNode(node) {
 
 
 /**
- * Creates a shadow root for an shadow embed.
- * @param {!Element} hostElement
- * @param {!Array<string>} extensionIds
- * @return {!ShadowRoot}
- */
-export function createShadowEmbedRoot(hostElement, extensionIds) {
```

Comment:
Long time ago. I didn't want to drag it around. Easy enough to restore if we'll decide it's useful again.